frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -0,0 +1,311 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import { IconButton, Checkbox } from 'tracim_frontend_lib'
+// import Form from '@rjsf/core'
+import Form from 'react-jsonschema-form-bs4'
+import TextareaRich from './TextareaRich.jsx'
+
+require('./CustomFormManager.styl')
+
+const DisplayTitle = props => {
+  if (!props.label) return null
+
+  return (
+    <div className='DisplayTitle'>
+      {props.label}
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyString = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span
+        className='DisplaySchemaPropertyString__value'
+        dangerouslySetInnerHTML={{ __html: props.value }}
+      />
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyBoolean = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span className='DisplaySchemaPropertyString__value'>
+        <Checkbox
+          name={props.label}
+          checked={props.value}
+          onClickCheckbox={() => {}}
+          styleLabel={{ top: '8px', cursor: 'default' }}
+          styleCheck={{ top: '-5px' }}
+        />
+      </span>
+    </div>
+  )
+}
+
+const DisplaySchemaArray = props => {
+  if (!props.valueList) return null
+
+  if (props.valueList.every(value => typeof value === 'string')) {
+    return (
+      <DisplaySchemaPropertyString
+        label={props.label}
+        value={props.valueList.join(', ')}
+      />
+    )
+  }
+
+  return [
+    <DisplayTitle
+      label={props.label}
+      key={props.label}
+    />,
+    props.valueList.map((value, i) => {
+      if (!value) return null
+
+      const valueType = typeof value
+
+      if (valueType === 'object') {
+        return (
+          <div
+            className='DisplaySchemaArray'
+            key={`object_${props.currentKey}_${value.title}_${i}`}
```

Comment:
```suggestion
            key={`${props.currentKey}${value.title}`}
```

Is value.title necessary?

Reply:
yes, because in the array I don't have a unique key for each elements. The "currentKey" is the parent key, which is always the same. that's why I concat the title and i

won't fix

Reply:
but that make me realize that `parentKey` would be better than `currentkey`
will do the renaming

Reply:
renaming done